packages/hash/api/src/graphql/resolvers/knowledge/model-mapping.ts

```diff
@@ -0,0 +1,31 @@
+import { EntityModel, PageModel } from "../../../model";
+import { KnowledgeEntity, KnowledgePage } from "../../apiTypes.gen";
+import { entityTypeModelToGQL } from "../ontology/model-mapping";
+
+export type ExternalEntityResolversGQL = "linkedEntities";
+export type UnresolvedEntityGQL = Omit<
+  KnowledgeEntity,
+  ExternalEntityResolversGQL
+>;
+
+export const entityModelToGQL = (
```

Comment:
For consistency with prior mapping functions:
```suggestion
export const mapEntityModelToGQL = (
```

Reply:
IMO yes

Reply:
Yes

Reply:
Adressed in 46e8b2ca96e2ba5de4be131c02367a710655e773 drive-by in b49e7eab8dc75e750115231ccfada6659af0997e

Reply:
Thanks :)